### PR TITLE
PMM-11080-pmm-cli-tests

### PR DESCRIPTION
**Pull request by yurkovychv**
https://jira.percona.com/browse/PMM-11080

- [ ] Links to other linked pull requests.

### CODE REVIEW

File: .github/workflows/pmm-cli.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    steps:
+      - name: checkout code
+        uses: percona-platform/checkout@v3
+        with:
+          submodules: 'recursive'
+
+      - name: Run ci.py
+        run: |
+          sudo apt-get install python3-setuptools
```
Comment (ademidoff) [inline .github/workflows/pmm-cli.yml]: Consider following the readme [here](https://github.com/Percona-Lab/pmm-submodules#installation-dependencies), which is `pip install...`

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2906-84c0916
client docker - perconalab/pmm-client-fb:PR-2906-84c0916
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-84c0916.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2906-84c0916&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-84c0916.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2906-866c16a
client docker - perconalab/pmm-client-fb:PR-2906-866c16a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-866c16a.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2906-866c16a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-866c16a.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2906-6087cb6
client docker - perconalab/pmm-client-fb:PR-2906-6087cb6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-6087cb6.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2906-6087cb6&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-6087cb6.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/11849/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2906-f4e87cb
client docker - perconalab/pmm-client-fb:PR-2906-f4e87cb
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-f4e87cb.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2906-f4e87cb&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-f4e87cb.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/11854/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2906-6c18d7f
client docker - perconalab/pmm-client-fb:PR-2906-6c18d7f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-6c18d7f.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2906-6c18d7f&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-6c18d7f.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/11856/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2906-401bfcd
client docker - perconalab/pmm-client-fb:PR-2906-401bfcd
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-401bfcd.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2906-401bfcd&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-401bfcd.tar.gz

**Review by ademidoff — APPROVED** [review]

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/11858/

**Review by BupycHuk — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2906-95fa322
client docker - perconalab/pmm-client-fb:PR-2906-95fa322
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-95fa322.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2906-95fa322&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-95fa322.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2906-c579577
client docker - perconalab/pmm-client-fb:PR-2906-c579577
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-c579577.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2906-c579577&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2906-c579577.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/12019/

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/12020/

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/11648/

**Review by talhabinrizwan — APPROVED** [review]

**Pull request merged**